### DISCUSSION TITLE
Advice for a web development....

### DISCUSSION THREAD

**Discussion by Rahul-Reddy-T** [community/community#126311] [Programming Help]
I wanted to learn and create a wab page or basically I wanted to do web application...can any one suggest how to start and what all to learn and how to pursue with my idea....

**Comment by ShahbaazX786** [comment]
Head over to chatgpt and ask the same question using the below prompt.

"Assume you are a 20+ years experienced developer, now tell me how to create a website like I'm 5."

**Comment by judahpaul16** [comment]
Just go to Harvard for free. https://youtube.com/playlist?list=PLhQjrBD2T380F_inVRXMIHCqLaNUd7bN4&si=WTsb8TO9mTmrzBDj
Lectures 8 and 9 handle web more specifically. All the slides, source code, and resources are [here](https://cs50.harvard.edu/x/2024/).

**Comment by goldieowner17** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @Rahul-Reddy-T ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. In this case - the Programming help category will be helpful.

Also, you are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!
